packages/client/src/views/RecordAudit/utils.ts

```diff
@@ -97,6 +97,17 @@ export const getFieldValue = (
   intl: IntlShape
 ) => {
   let original = value
+  // HotFix: The fieldObject that are receiving here is internationStatePrimary
+  // rather than statePrimary same for district
```

Comment:
```suggestion
  // HOTFIX: The name of the fieldObject that is being received
  // here is internationalStatePrimary rather than statePrimary
  // same for districts as well as the secondary address fields
```